packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -407,16 +630,26 @@ fn array_to_user_errors(array: mv8::Value<'_>) -> Vec<UserError> {
     vec![UserError(fallback)]
 }
 
-fn array_to_user_warnings(array: mv8::Value<'_>) -> Vec<UserWarning> {
+fn array_to_user_warnings<'s>(
+    scope: &mut v8::HandleScope<'s>,
+    array: v8::Local<'s, v8::Value>,
+) -> Vec<UserWarning> {
     // TODO: Extract optional line numbers
-    let fallback = format!("Unparsed: {:?}", array);
-
-    if let mv8::Value::Array(array) = array {
-        let warnings = array
-            .elements()
-            .map(|e: mv8::Result<'_, mv8::Value<'_>>| {
-                e.map(|e| UserWarning {
-                    message: format!("{:?}", e),
+    let fallback = format!("Unparsed: {array:?}");
+
+    if array.is_array() {
```

Comment:
Why not pass `Array` to this function directly?

Reply:
It was the previous behavior, in case it's not an `Array` it's ignored.

Reply:
Yes, but do we want to silently ignore this, if the expected type is an array? (The function has `array` in it's name)

Reply:
I mean... I don't know if it's ok or not, is this behavior documented somewhere for users?

We could rename the function to `value_to_user_errors` but at the same time, only arrays are a valid input.